### ISSUE TITLE
Could not create Azure arc enabled kubernetes logic app on vscode

### ISSUE THREAD

**Issue by AliRz90** [Azure/LogicAppsUX#5970]
Does this occur consistently? 

 Repro steps:

1\.

 2\.

Action: azureLogicAppsStandard.createLogicAppAdvanced

 Error type: Error

 Error Message: Error in creating logic app. RestError: The request content was invalid and could not be deserialized: 'Required property 'type' not found in JSON. Path 'extendedLocation', line 1, position 309.'.

Version: 4.107.3

 OS: win32

 OS Release: 10.0.22631

 Product: Visual Studio Code

 Product Version: 1.94.2

 Language: en

<details><summary>Call Stack</summary>¨NBSP;<p></p><precode language="" precodenum="0"></precode><p></p></details>

**Comment by rllyy97** [comment]
Hi @AliRz90, it looks like you have an error with your workflow code, could you please share your workflow JSON?  you might just need to add the "type" property your error message is mentioning.

**Comment by AliRz90** [comment]
I wanted to create azure arc enabled logic app via vscode, which I couldn't. I have also tried to do that with azure cli, and the extension won't be install  az extension add --yes --source "https://aka.ms/logicapp-latest-py2.py3-none-any.whl" and get this error "An error occurred. Pip failed with status code 1. Use --debug for more information.", then I tried IaC, which finally I could successfully deploy the logic app, however it's not working.
